integrations/Braze/browser.js

```diff
@@ -42,71 +43,33 @@ class Braze {
+      y.src='https://js.appboycdn.com/web-sdk/4.2/braze.min.js';
+      y.async=1;(b=p.getElementsByTagName(P)[0]).parentNode.insertBefore(y,b)
+    }(window,document,'script');
 
-    window.appboy.initialize(this.appKey, {
+    window.braze.initialize(this.appKey, {
       enableLogging: true,
       baseUrl: this.endPoint,
```

Comment:
We are removing these two functionality. Are those not supported with the new SDK? If these can't be supported we have to remove these from our dashboard and docs.

Reply:
which 2 functionalities ? only avatar support is removed